gradle.properties

```diff
@@ -15,5 +19,3 @@ developers=Graeme Rocher
 
 githubCoreBranch=3.2.x
 bomProperty=micronautRxJava3Version
-bomProperties=rxJava3Version
```

Comment:
@sdelamo I removed `bomProperties` Just checking this is correct?

Reply:
Yes, I think it is correct